### PR TITLE
feat: add galxe banners

### PR DESCRIPTION
**Pull request by ChefBingbong**
### Summary
🎁🎂🚀

Added two new banners to the home page to promote the Galaxe Predict and Galaxe Syndicate features, which are prediction and perpetual trading games with $10,000 rewards. Created new components, images, and translations for the banners.

> _Sing, O Muse, of the glorious banners that adorn the pancakeswap app_
> _The work of skillful coders who crafted `GalaxePredictBanner.tsx` and `GalaxeSyndicateBanner.tsx`_
> _To celebrate the birthday of Galaxe, the mighty planet of the gods_
> _And to entice the users with rewards, like golden apples of the Hesperides_

### Walkthrough
*  Add two new banner components for the Jupiter Predictors and Jupiter Syndicate campaigns ([link](https://github.com/pancakeswap/pancake-frontend/pull/8026/files?diff=unified&w=0#diff-a09a3f739ca3251a7221820b4749b8c7e80df7a6608978a99a4f3b4af56e626eR1-R177), [link](https://github.com/pancakeswap/pancake-frontend/pull/8026/files?diff=unified&w=0#diff-eb23a738420ff247df277feeb802c174fa426b16dee440e7062fec5cefed6e0cR1-R178))
*  Import the new components and add them to the banner configuration hook ([link](https://github.com/pancakeswap/pancake-frontend/pull/8026/files?diff=unified&w=0#diff-214af478b927b76b48361f0a7ba597c535ab140dcf4e7281bf15abde2a1a0e89R19-R20), [link](https://github.com/pancakeswap/pancake-frontend/pull/8026/files?diff=unified&w=0#diff-214af478b927b76b48361f0a7ba597c535ab140dcf4e7281bf15abde2a1a0e89R62-R64))
*  Import and export the images used by the new components from `images/index.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/8026/files?diff=unified&w=0#diff-0ede1236551332d6554652d9afbf09aceeb9d948aa530efb754e7428f09adbeeL88-R96), [link](https://github.com/pancakeswap/pancake-frontend/pull/8026/files?diff=unified&w=0#diff-0ede1236551332d6554652d9afbf09aceeb9d948aa530efb754e7428f09adbeeR112-R113))
*  Add the localized texts for the new components to `translations.json` ([link](https://github.com/pancakeswap/pancake-frontend/pull/8026/files?diff=unified&w=0#diff-eb2ab983e2cefc22516cb7814c86346f4b2bd2b971980952868173095d48f459L2802-R2806))

### CODE REVIEW

**Comment by ChefMomota** [comment]
Take down Polygon, zksync era, and ARB banner.

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx [hunk 1]
```diff
@@ -57,6 +59,9 @@ export const useMultipleBannerConfig = () => {
     const SHUFFLE_BANNERS: IBannerConfig[] = [
       { shouldRender: true, banner: <LineaBanner /> },
       { shouldRender: true, banner: <ArbitrumOneBanner /> },
+      { shouldRender: true, banner: <GalxePedictBanner /> },
+      { shouldRender: true, banner: <GalxeSyndicateBanner /> },
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:63]: Move both to `NO_SHUFFLE_BANNERS ` array, and make them before `<UserBanner/> `.

**Comment by ChefMomota** [comment]
Fix this
Screenshot 2023-10-03 at 8 42 57 AM

**Comment by ChefMomota** [comment]
Screenshot 2023-10-03 at 8 49 39 AM
Screenshot 2023-10-03 at 8 49 46 AM

File: apps/web/src/views/Home/components/Banners/GalxeSyndicateBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+            >
+              {isMobile ? t('Saturn Syndicate') : t('Daily reward for new users and to top the leaderboard')}
+            </StyledSubheading>
+          </Box>
+
+          <Flex marginTop={isMobile ? '24px' : '12px'}>
+            <Link
+              style={{ textDecoration: 'none' }}
+              external
+              href="https://perp.pancakeswap.finance/en/futures/v2/BTCUSD?theme=light"
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/GalxeSyndicateBanner.tsx:123]: use `getPerpetualUrl`

File: apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+              src={pancakeSwapLogo}
+              alt="pancakeSwapLogo"
+              width={isMobile ? 90 : 132}
+              height={isMobile ? 13 : 20}
+              unoptimized
+            />
+          </Flex>
+          <Box maxWidth="780px" marginTop="12px">
+            {!isMobile ? (
+              <StyledSubheading data-text={t('Jupiter Predictors')}>{t('Jupiter Predictors:')}</StyledSubheading>
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx:108]: <StyledSubheading data-text={t('Jupiter Predictors')}>{t('Jupiter Predictors:')}</StyledSubheading>

to be

<StyledSubheading data-text={t('Jupiter Predictors:')}>{t('Jupiter Predictors:')}</StyledSubheading>

Comment (ChefMomota) [reply apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx:108]: We can always show this right?

File: apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+            ) : null}
+            <StyledSubheading
+              data-text={isMobile ? t('Jupiter Predictors') : t('Predict, win and share $10,000 rewards')}
+            >
+              {isMobile ? t('Jupiter Predictors') : t('Predict, win and share $10,000 rewards')}
+            </StyledSubheading>
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx]: If upper always show Jupiter Predictors. I don't think we need show two time.
`Jupiter Predictors`

File: apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx [hunk 5]
```diff
@@ -55,10 +55,10 @@ export const useMultipleBannerConfig = () => {
...
 
     const SHUFFLE_BANNERS: IBannerConfig[] = [
-      { shouldRender: true, banner: <LineaBanner /> },
-      { shouldRender: true, banner: <ArbitrumOneBanner /> },
-      { shouldRender: true, banner: <ZksyncBanner /> },
-      { shouldRender: true, banner: <PolygonZkEvmBanner /> },
+      // { shouldRender: true, banner: <LineaBanner /> },
+      // { shouldRender: true, banner: <ArbitrumOneBanner /> },
+      // { shouldRender: true, banner: <ZksyncBanner /> },
+      // { shouldRender: true, banner: <PolygonZkEvmBanner /> },
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx:61]: Remove it

File: apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+          </Flex>
+        </S.LeftWrapper>
+        <RightWrapper>
+          <BgWrapper>
+            {!isMobile ? (
+              <Image src={galxePredictorsBg} height={192} alt="liquidStakingBunny" placeholder="blur" />
+            ) : (
+              <Image src={galxeBirthdayCampaignPedictMobile} height={192} alt="liquidStakingBunny" placeholder="blur" />
+            )}
+          </BgWrapper>
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx:161]: Size weird. add width.
And use x2 image

File: apps/web/src/views/Home/components/Banners/GalxeSyndicateBanner.tsx [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+            </Link>
+          </Flex>
+        </S.LeftWrapper>
+        <RightWrapper>
+          <BgWrapper>
+            {!isMobile ? (
+              <Image src={galxeSyndicateBg} height={222} alt="liquidStakingBunny" placeholder="blur" />
+            ) : (
+              <Image src={galxeBirthdayCampaignPerpMobile} height={192} alt="liquidStakingBunny" placeholder="blur" />
+            )}
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/GalxeSyndicateBanner.tsx:171]: add width

File: apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx (hunk 6 above)
Comment (ChefBingbong) [reply apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx]: width added

File: apps/web/src/views/Home/components/Banners/GalxeSyndicateBanner.tsx (hunk 7 above)
Comment (ChefBingbong) [reply apps/web/src/views/Home/components/Banners/GalxeSyndicateBanner.tsx]: and here too

File: apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+          </Flex>
+          <Box maxWidth="780px" marginTop="12px">
+            {!isMobile ? (
+              <StyledSubheading data-text={t('Jupiter Predictors')}>{t('Jupiter Predictors')}</StyledSubheading>
+            ) : null}
+            <StyledSubheading
+              data-text={isMobile ? t('Jupiter Predictors:') : t('Predict, win and share $10,000 rewards')}
+            >
+              {isMobile ? t('Jupiter Predictors:') : t('Predict, win and share $10,000 rewards')}
+            </StyledSubheading>
```
Comment (ChefMomota) [inline apps/web/src/views/Home/components/Banners/GalxePedictBanner.tsx:116]: we can change to this.  We always show `Jupiter Predictors`

<StyledSubheading data-text={t('Jupiter Predictors')}>{t('Jupiter Predictors')}</StyledSubheading>
            {!isMobile && (
              <StyledSubheading data-text={t('Predict, win and share $10,000 rewards')}>
              {t('Predict, win and share $10,000 rewards')}
              </StyledSubheading>
            )}

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**